sdks/java/nexmark/build.gradle

```diff
@@ -42,6 +42,7 @@ configurations {
 }
 
 dependencies {
+  compile 'com.google.cloud:google-cloud-bigquery:1.28.0'
```

Comment:
We don't need this anymore, right ? Also, this can result in dependency conflicts, see https://issues.apache.org/jira/browse/BEAM-4229.

Reply:
+1, duplicate comment, see above.